src/kaocha/testable.clj

```diff
@@ -174,7 +189,8 @@
                  :kaocha/testable test})
             m (cond-> m
                 file (assoc :file file)
-                line (assoc :line line))]
+                line (assoc :line line)
+                thread (assoc :thread thread))]
```

Comment:
We might want to namespace this? things like type/actual/expected/line/file are standard clojure.test things, if we shove extra data into reporter events we generally namespace them. Not sure how consistent we are in this but generally a good idea.